source/blog/2018-08-10-the-ember-times-issue-59.md

```diff
@@ -31,8 +31,28 @@ You can already test this by [getting the beta](https://github.com/ember-cli/emb
 
 ---
 
-## [SECTION TITLE](#section-url)
-
+## [Knowing Boundaries with the Component Manager Bounds RFC](https://github.com/emberjs/rfcs/pull/351)
+
+A new Request for Comments (RFC) is aiming to improve the flexibility of another previously
+RFC'ed feature that is interesting for Ember addon authors who are passionate about `Components`.
+The [Component Manager™ feature](https://emberjs.github.io/rfcs/0213-custom-components.html) which had
+already had been accepted by the Ember community in the spring of this year ([xxx](https://github.com/emberjs/rfcs/pull/213#issuecomment-374981569)) defined a **new low-level primitive**
```

Comment:
maybe not `xxx` just `emberjs/rfcs/pull/213` or something?

Reply:
Totally, missed that one, good catch :+1: